pilot/pkg/features/pilot.go

```diff
@@ -415,7 +415,14 @@ var (
 	)
 
 	TrustedGatewayCIDR = func() []string {
-		return strings.Split(trustedGatewayCIDR.Get(), ",")
+		cidr := trustedGatewayCIDR.Get()
+
+		// splitting the empty string will result [""]
+		if cidr == "" {
+			return []string{}
+		}
+
+		return strings.Split(cidr, ",")
```

Comment:
```suggestion
		if cidr,f := trustedGatewayCIDR.lookup(); f {
		  return strings.Split(trustedGatewayCIDR.Get(), ",")
		}
		return nil
```

Slightly more clear and avoids an (extraordinarily unlikely) race condition that the Get changes between calls

Reply:
oh actually you already fixed that